src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -50,7 +50,13 @@ def __call__(
         self.text_encoder.to(torch_device)
 
         # get prompt text embeddings
-        text_input = self.tokenizer(prompt, padding=True, truncation=True, return_tensors="pt")
+        text_input = self.tokenizer(
+            prompt,
+            padding="max_length",
+            max_length=self.tokenizer.model_max_length,
+            truncation=True,
+            return_tensors="pt",
+        )
```

Comment:
This is important here to always pad to max_length, as that's how the model was trained.

Reply:
yes agree